bootloaders/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+add_subdirectory_exclude_platforms(uart host rp2040)
```

Comment:
I guess it's the use of "partitions" (rather than anything else) that means that the `bootloaders/uart` example won't work on RP2040? But in theory at least (if you used some kind of other "enclosed binary" mechanism), there's no reason that RP2040 couldn't UART-boot an RP2350?

Reply:
Yes, that's correct